### PR TITLE
Load missing lang vars for SCEditor

### PR DESCRIPTION
**Pull request by dragomano**
We do not have translations for some elements of SCEditor's toolbar.

in Russian:
2020-11-21 06_13_40-Id nihil adipisci ipsum  — Firefox Developer Edition
in Spanish:
2020-11-21 06_13_16-Id nihil adipisci ipsum  — Firefox Developer Edition

After this fix
in Russian::
2020-11-21 06_14_02-Id nihil adipisci ipsum  — Firefox Developer Edition
in Spanish:
2020-11-21 06_14_17-Id nihil adipisci ipsum  — Firefox Developer Edition

Signed-off-by: Bugo <bugo@dragomano.ru>

### CODE REVIEW

**Comment by live627** [comment]
Thanks for all your work so far. 😃 

While testing this I noticed the email and URL popups are still English, as well as the more smileys option and the close button within said popup.

File: Sources/Subs-Editor.php [hunk 1]
```diff
@@ -1563,6 +1523,25 @@ function create_control_richedit($editorOptions)
 		loadJavaScriptFile('editor.js', array('minimize' => true), 'smf_editor');
 		loadJavaScriptFile('jquery.sceditor.bbcode.min.js', array(), 'smf_sceditor_bbcode');
 		loadJavaScriptFile('jquery.sceditor.smf.js', array('minimize' => true), 'smf_sceditor_smf');
+
+		$scExtraLangs = '
+		$.sceditor.locale["' . $txt['lang_dictionary'] . '"] = {
+			"Width (optional):": "' . $editortxt['width'] . '",
+			"Height (optional):": "' . $editortxt['height'] . '",
+			"Insert": "' . $editortxt['insert'] . '",
+			"Description (optional):": "' . $editortxt['description'] . '",
```
Comment (live627) [inline Sources/Subs-Editor.php:1532]: Needs to be defined in the language file (Editor.english.php)

File: Sources/Subs-Editor.php [hunk 2]
```diff
@@ -1563,6 +1523,25 @@ function create_control_richedit($editorOptions)
...
+			"Rows:": "' . $editortxt['rows'] . '",
+			"Cols:": "' . $editortxt['cols'] . '",
+			"URL:": "' . $editortxt['url'] . '",
+			"E-mail:": "' . $editortxt['email'] . '",
+			"Video URL:": "' . $editortxt['video_url'] . '",
+			"More": "' . $editortxt['more'] . '",
+			"Close": "' . $editortxt['close'] . '",
+			dateFormat: "' . $editortxt['dateformat'] . '"
```
Comment (live627) [inline Sources/Subs-Editor.php:1540]: Needs to be defined in the language file (Editor.english.php)

**Review by live627 — CHANGES_REQUESTED** [review]

File: Sources/Subs-Editor.php (hunk 2 above)
Comment (dragomano) [reply Sources/Subs-Editor.php:1540]: What exactly?

**Comment by live627** [comment]
$editortxt['dateformat']

$editortxt['description']

Maybe they could be in the error log?

**Comment by dragomano** [comment]
> $editortxt['dateformat'] $editortxt['description'] Maybe they could be in the error log?
But they [already are added](https://github.com/SimpleMachines/SMF2.1/pull/6362/commits/2034ad57e6ad7cc0a68dfa17d6f8c5ea46798f4c) in Editor.english.php, aren't they?

**Review by live627 — APPROVED** [review]
Yo, I didn't notice that and therefore didn't update that on my end. Sorry.

**Pull request merged**